lib/compiler.js

```diff
@@ -2,7 +2,7 @@
 (function() {
   var CoffeeScript, indent, precompile, preprocess;
 
-  CoffeeScript = require("coffee-script");
+  CoffeeScript = require("coffeescript");
```

Comment:
I was led to doing all this from seeing the CI issue off bigmaven:

![image](https://user-images.githubusercontent.com/142403/60475480-67792f00-9c2c-11e9-8563-d7b8e1767cfb.png)